tito.8.asciidoc

```diff
@@ -101,6 +101,11 @@ Automatically accept the generated changelog.
 Use 'MESSAGE' as the default changelog message for new
 packages
 
+--changelog-message='MESSAGE'::
+Use 'MESSAGE' as the changelog when tagging, this can be specified
+everytime a package is tagged. Useful when needing to generate
+custom changelogs.
+
```

Comment:
Could you clarify this overrides the auto-generated changelog, and optionally if you'd prefer --changelog as well, update throughout. (I'll go either way)